src/page.njk

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    {{ headFileTopContent }}
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="generator" content="{{ markBindVersion }}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ title }}</title>
+    <link rel="stylesheet" href="{{ asset.bootstrap }}">
+    <link rel="stylesheet" href="{{ asset.bootstrapVue }}">
+    <link rel="stylesheet" href="{{ asset.fontAwesome }}">
+    <link rel="stylesheet" href="{{ asset.glyphicons }}">
+    <link rel="stylesheet" href="{{ asset.highlight }}">
+    <link rel="stylesheet" href="{{ asset.markbind }}">
+    {% if asset.pluginLinks %}
+        {% for link in asset.pluginLinks %}
+            {{ link }}
+        {% endfor %}
+    {% endif %}
```

Comment:
There are now a lot of blank spaces in the output.

```html
...
  <link rel="stylesheet" href="markbind/fontawesome/css/all.min.css">
  <link rel="stylesheet" href="markbind/glyphicons/css/bootstrap-glyphicons.min.css">
  <link rel="stylesheet" href="markbind/css/github.min.css">
  <link rel="stylesheet" href="markbind/css/markbind.css">



  <link rel="stylesheet" href="markbind/layouts/default/styles.css">

  <link rel="stylesheet" href="markbind/css/site-nav.css">


  <link rel="stylesheet" href="markbind/css/page-nav.css">



</head>
```

You can get rid of the blank spaces in the render by using [whitespace control](https://mozilla.github.io/nunjucks/templating.html#whitespace-control). For example, changing the above to this will (1) remove unnecessary whitespace, and (2) make the <link> align with each other:

```
{%- if asset.pluginLinks %}
{%- for link in asset.pluginLinks %}
    {{ link }}
{%- endfor %}
{%- endif %}
```

Reply:
The format and indentation in the original file `src/page.ejs` are like that due to the whitespace issue, so if unsure following the original one is the easiest.

Reply:
I applied the whitespace control to the entire file. Thanks for the tips!